signing/transaction-manager.js

```diff
@@ -82,7 +83,10 @@ export async function createSignBroadcast({
       [].concat(messages),
       {
         amount: transactionData.fee,
-        gas: transactionData.gasEstimate,
+        gas:
+          messageType === 'ClaimRewardsTx'
+            ? transactionData.gasEstimate * validators.length
+            : transactionData.gasEstimate,
```

Comment:
Can we write a switch case instead of ternary operator? easier to expand later on
Also we can abstract `validator` to something like `gasEstimateMultiplier` which is `validators.length` in this case, so that we can reuse this for multi-send or other tx.